packages/contracts/package.json

```diff
@@ -26,6 +27,7 @@
         "lint": "tslint --project . --exclude **/src/contract_wrappers/**/* --exclude **/lib/**/*",
         "coverage:report:text": "istanbul report text",
         "coverage:report:html": "istanbul report html && open coverage/index.html",
+        "profiler:report:html": "istanbul report html && open coverage/index.html",
```

Comment:
Hm... so both coverage and profiler write to `coverage/index.html`? Any chance we could split them up? Otherwise they will overwrite one another, no?

Reply:
Yes. I'm aware of that. Unfortunately, Istanbul has no way to specify the file name/path. That's why I have a check that coverage/profiling are mutually exclusive.